test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAProjectWithDependencies.cs

```diff
@@ -147,9 +149,11 @@ public void It_publishes_projects_with_simple_dependencies_with_filter_profile()
             });
 
            var runtimeConfig = ReadJson(System.IO.Path.Combine(publishDirectory.ToString(), $"{project}.runtimeconfig.json"));
-
            runtimeConfig["runtimeOptions"]["tfm"].ToString().Should().Be("netcoreapp1.1");
-            
+
+            var depsJson = ReadJson(System.IO.Path.Combine(publishDirectory.ToString(), $"{project}.deps.json"));
+            depsJson["libraries"]["Newtonsoft.Json/9.0.1"]["runtimeStoreManifestName"].ToString().Should().Be($"{manifestFileName1} {manifestFileName2}");
```

Comment:
Should have an end-to-end test that ensures when this package can't be found (because the runtime store is unavailable), that a good exception is displayed.